automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala

```diff
@@ -29,7 +29,7 @@ class WorkspaceWriterSpec extends FreeSpec with Matchers with WebBrowserSpec wit
       "should only see the estimated monthly storage fee in the Project Cost section of the summary page" in {
         val user = UserPool.chooseStudent
         implicit val authToken: AuthToken = authTokenOwner
-        val testName = "WorkspaceWriterSpec_writerAccess_projectCost"
+        val testName = "WorkspaceWriterAccess"
```

Comment:
use shorter string for workspace name. long string doesn't add any benefit and harder to read in test log.